doc/source/community/contributing.rst

```diff
@@ -147,23 +147,23 @@ after updating.
 
 2) Creating a development environment
 ---------------------------------------
-A development environment is a virtual space where you can keep an independent installation of *GeoPandas*.
+A development environment is a virtual space where you can keep an independent installation of GeoPandas.
 This makes it easy to keep both a stable version of python in one place you use for work, and a development
 version (which you may break while playing with code) in another.
 
-An easy way to create a *GeoPandas* development environment is as follows:
+An easy way to create a GeoPandas development environment is as follows:
 
 - Install either `Anaconda <http://docs.continuum.io/anaconda/>`_ or
   `miniconda <http://conda.pydata.org/miniconda.html>`_
 - Make sure that you have :ref:`cloned the repository <contributing.forking>`
-- ``cd`` to the *geopandas** source directory
+- ``cd`` to the GeoPandas source directory
```

Comment:
This one may be referring to the `geopandas` folder rather than a project name or not?